samples/portability/cmsis_rtos_v2/timer_synchronization/src/main.c

```diff
@@ -59,7 +59,9 @@ void send_msg_thread(void)
 		printk("Wrote to message queue: %d\n", data);
```

Comment:
Move the return 0 here and then take out the "else".

Reply:
@rgundi: Done, Thanks

Reply:
@rgundi: Done, Thanks